ostelco-core/Models/MyInfoDetails.swift

```diff
@@ -137,7 +137,7 @@ public struct MyInfoDetails: Codable {
             _regadd = newValue
         }
     }
-    
+        
     public let mobileNumber: MyInfoMobileNumber?
```

Comment:
This needs to go away. We don't collect mobile number anymore, so it'll be confusing to see this.